src/HotChocolate/Spatial/src/Types.Spatial/GeoJSONLineStringType.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using HotChocolate;
+using HotChocolate.Types;
+using NetTopologySuite.Geometries;
+
+namespace Types.Spatial
+{
+    public class GeoJSONLineStringType : ObjectType<LineString>
+    {
+        protected override void Configure(IObjectTypeDescriptor<LineString> descriptor)
+        {
+            descriptor.BindFieldsExplicitly();
+
+            descriptor.Implements<GeoJSONInterface>();
+
+            descriptor.Field("type").Resolver(GeoJSONGeometryType.LineString);
```

Comment:
This could be either better done with a proper shared resolver or with `ResolveWith`